lib/index.js

```diff
@@ -162,6 +165,76 @@ export default {
     return provider
   },
 
+  provideHyperclick(): HyperclickProvider {
```

Comment:
IMO remove the `flowtype` plugin and either
* declare this type
* or replace it with `Object` like it is done for autocomplete or the linter provider

same for `HyperclickSuggestion` below